### PR TITLE
Fixes #31466: Switch HighLine back to using :info color

### PR DESCRIPTION
**Pull request by ehelms**
HighLine does not have a :notice color which was changed as part
of the switchover from info to notice as the default log level.

### CODE REVIEW

**Review by ekohl — COMMENTED** [review]
Should we add some tests to forklift that calls foreman-installer --list-scenarios to catch these regressions in the future?

**Comment by ehelms** [comment]
I pushed a test that handles checking it right here in the Kafo code base.

File: test/acceptance/kafo_configure_test.rb [hunk 1]
```diff
@@ -52,6 +52,15 @@ module Kafo
       end
     end
 
+    describe '--list-scenarios' do
+      it 'lists scenarios' do
+        code, out, err = run_command '../bin/kafo-configure --list-scenarios'
+        _(code).must_equal 0, err
+        _(out).must_include "Available scenarios"
+        _(out).must_include "default \e[0m(use: --scenario default)\n"
```
Comment (evgeni) [inline test/acceptance/kafo_configure_test.rb:60]: seems the tests don't use colors and fail on the difference.

Comment (ehelms) [reply test/acceptance/kafo_configure_test.rb]: Darn dynamic calculation based on environment. I set it to `--no-colors` to be explicit.

**Review by evgeni — APPROVED** [review]

**Pull request merged**